x-pack/test/functional/apps/upgrade_assistant/deprecation_pages.ts

```diff
@@ -64,7 +62,7 @@ export default function upgradeAssistantFunctionalTests({
         });
 
         // Wait for the cluster settings to be reflected to the ES nodes
-        await setTimeout(30000);
+        await setTimeout(12000);
```

Comment:
Has the test changes been run through the flaky test runner to verify this is enough time?

Reply:
I am able to reproduce the flakiness locally and this timeout was more than enough